docs/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-repositories.md

```diff
@@ -67,7 +68,7 @@ To view Helm charts in the OCI-based Helm chart repository after it achieves an
 
 ## Refresh an OCI-Based Helm Chart Repository
 
-Rancher automatically refreshes the OCI-based Helm chart repository every 6 hours. 
+Rancher automatically refreshes the OCI-based Helm chart repository every 24 hours but the user has the option to change it.
```

Comment:
```suggestion
By default, Rancher automatically refreshes the OCI-based Helm chart repository every 24 hours. This can be modified by setting the **Refresh Interval** field.
```